spec/controllers/idv/how_to_verify_controller_spec.rb

```diff
@@ -221,6 +221,18 @@
 
         expect(@analytics).to have_logged_event(analytics_name, analytics_args)
       end
+
+      context 'the user has an establishing in-person enrollment' do
+        let(:user) { create(:user, :with_establishing_in_person_enrollment) }
+
+        it 'cancels the in-person enrollment' do
+          expect(user.in_person_enrollments.first.status).to eq('establishing')
+
+          put :update, params: params
+
+          expect(user.in_person_enrollments.first.status).to eq('cancelled')
```

Comment:
This is purely a matter of opinion; please feel free to wholly disregard it. But I wonder if this would read better as:

```ruby
expect { put :update, params: params }.to change { user.in_person_enrollments.first.status }
 .from('establishing')
 .to('cancelled')
```

I like the clarify of `.to change ... from ... to`, but with multiple blocks maybe it's not actually clearer in this case.

Reply:
See [a1ed99f](https://github.com/18F/identity-idp/pull/11948/commits/a1ed99fc573e6034c7762b018deda884e5d91921).